kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSValueArgumentImpl.kt

```diff
@@ -25,6 +25,7 @@ class KSValueArgumentImpl(private val namedAnnotationValue: KtNamedAnnotationVal
         KSNameImpl(namedAnnotationValue.name.asString())
     }
     override val isSpread: Boolean = false
+    override val isDefault: Boolean = false
```

Comment:
I'm not sure if this is correct. I don't understand what this module is used for or where the data comes from

Reply:
You can put a TODO() here, this is a WIP development module.